packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -104,6 +110,12 @@ class WebKitWebViewControllerCreationParams
   /// Defaults to false.
   final bool allowsInlineMediaPlayback;
 
+  /// Whether to limit navigation to configured domains. (Only available for iOS > 14.0)
```

Comment:
This should separate the first sentence: https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph